spec/system/devise_spec.rb

```diff
@@ -25,12 +25,15 @@
       visit "/events.json"
     end
 
+    scenario "unauthenticated users can not delete events" do
+      visit "/events"
+      expect(page).not_to have_link "Delete"
+    end
+
     scenario "only authenticated users can delete events" do
+      sign_in FactoryBot.create(:admin), scope: :user
       visit "/events"
-      accept_alert do
```

Comment:
This test was failing locally and not in CI.  I do not think this was due to the upgrade, but I fixed it anyway.